content/en/docs/quick-start/enable-pluggable-components.md

```diff
@@ -59,17 +59,17 @@ If you adopt [All-in-one Installation](../../quick-start/all-in-one-on-linux/),
 
 ### Installing on Kubernetes
 
-When you install KubeSphere on Kubernetes, you need to download the file [cluster-configuration.yaml](https://github.com/kubesphere/ks-installer/releases/download/v3.0.0/cluster-configuration.yaml) for cluster setting. If you want to install pluggable components, do not use `kubectl apply -f` directly for this file.
+When you install KubeSphere on Kubernetes, you need to download the file [cluster-configuration.yaml](https://github.com/kubesphere/ks-installer/releases/download/v3.0.0/cluster-configuration.yaml) for cluster setting.
 
-1. In the tutorial of [Installing KubeSphere on Kubernetes](../../installing-on-kubernetes/introduction/overview/), you execute `kubectl apply -f` first for the file [kubesphere-installer.yaml](https://github.com/kubesphere/ks-installer/releases/download/v3.0.0/kubesphere-installer.yaml). After that, to enable pluggable components, create a local file cluster-configuration.yaml.
+1. To enable pluggable components you want to install, change `false` to `true` for `enabled` in the file cluster-configuration.yaml. Here is [an example file](https://github.com/kubesphere/ks-installer/blob/release-3.0/deploy/cluster-configuration.yaml) for your reference. Save the file after you finish..
 
 ```bash
 vi cluster-configuration.yaml
 ```
 
-2. Copy all the content in the file [cluster-configuration.yaml](https://github.com/kubesphere/ks-installer/releases/download/v3.0.0/cluster-configuration.yaml) and paste it to the local file just created.
-3. In this local cluster-configuration.yaml file, enable the pluggable components you want to install by changing `false` to `true` for `enabled`. Here is [an example file](https://github.com/kubesphere/ks-installer/blob/master/deploy/cluster-configuration.yaml) for your reference. Save the file after you finish.
-4. Execute the following command to start installation:
+2. In the tutorial of [Installing KubeSphere on Kubernetes](../../installing-on-kubernetes/introduction/overview/), you execute `kubectl apply -f` first for the installer file [kubesphere-installer.yaml](https://github.com/kubesphere/ks-installer/releases/download/v3.0.0/kubesphere-installer.yaml) which waits the file cluster-configuration.yaml to apply for continuing the installation.
```

Comment:
OK, I see what you mean, but my concern is that if you tell readers to download the cluster-configuration.yaml file and edit it in the first step, they may forget to apply the KubeSphere-installer.yaml file first. Namely, they may edit and apply the cluster-configuration.yaml file first.

How about this:

## Installing on Kubernetes

When you install KubeSphere on Kubernetes, you need to execute `kubectl apply -f` first for the installer file [kubesphere-installer.yaml](https://github.com/kubesphere/ks-installer/releases/download/v3.0.0/kubesphere-installer.yaml) as stated in the tutorial of [Installing KubeSphere on Kubernetes](../../installing-on-kubernetes/introduction/overview/). After that, follow the steps below to enable pluggable components:

1. Download the file [cluster-configuration.yaml](https://github.com/kubesphere/ks-installer/releases/download/v3.0.0/cluster-configuration.yaml) and copy and paste the content of it to a local `cluster-configuration.yaml` file.

```bash
vi cluster-configuration.yaml
```

2. To enable the pluggable component you want to install, change `false` to `true` for `enabled` under the component in this file.
3. Save this local file and execute the following command to apply it.

```bash
kubectl apply -f cluster-configuration.yaml
```

{{< notice warning >}} 

You must apply the `kubesphere-installer.yaml` file first before you edit and apply the file `cluster-configuration.yaml`. Wrong execution order or the failure to apply either file can result in installation failure.

{{</ notice >}}

Reply:
I am OK with this.